### ISSUE TITLE
terraform.py don't returns public IP's

### ISSUE THREAD

**Issue by altvnk** [CiscoCloud/kubernetes-ansible#35]
When using terraform deployment with floating ip's terraform.py don't returns public ip's and returns private instead.

**Comment by kenjones-cisco** [comment]
It is due to the fact that the MI version has changes that were not applied to the version within project terraform.py.

```
    if 'floating_ip' in raw_attrs:
        attrs['private_ipv4'] = raw_attrs['network.0.fixed_ip_v4']
```

https://github.com/CiscoCloud/microservices-infrastructure/commit/37baa1067bbc9ccb158e9ddf05af6cd3e521bd3d#diff-6a5dab50836db01e702f478b8820b7c1

**Comment by altvnk** [comment]
Sure, i just leaved it here as a task to be done
How do you think, we cleaned out all third-party cloud providers, maybe it's reasonable to clean `terraform.py` also?

**Comment by kenjones-cisco** [comment]
Ideally, we would leverage the terraform.py project but as it was feared when first thought of and done, it would get out of sync without due diligence.

But ideal and reality are not always friends; making the above change locally and even dropping the extra stuff does work since we dropped everything else that was similar to MI.

**Comment by altvnk** [comment]
Well, currently there is nothing to do with MI, so we free to decide.

**Comment by kenjones-cisco** [comment]
Created PR to resolve the current issue, and then we can determine if it makes sense long term to strip out the "extras" from it and just maintain our own.

**Comment by altvnk** [comment]
Ref #37

**Comment by kenjones-cisco** [comment]
Created #38 that strips terraform.py down to the only what is required to support this project and the cloud provider openstack.

**Comment by kenjones-cisco** [comment]
Looks like all items are covered, ready to close?

**Comment by altvnk** [comment]
Sure. Closed via #38
